simple_ado/wiki.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""ADO Wiki API wrapper."""
+
+import logging
+
+from simple_ado.base_client import ADOBaseClient
+from simple_ado.http_client import ADOHTTPClient, ADOResponse
+
+
+class ADOWikiClient(ADOBaseClient):
+    """Wrapper class around the ADO Wiki APIs.
+
+    :param http_client: The HTTP client to use for the client
+    :param log: The logger to use
+    """
+
+    def __init__(self, http_client: ADOHTTPClient, log: logging.Logger) -> None:
+        super().__init__(http_client, log.getChild("wiki"))
+
+    def get_page_version(self, page_id: str, wiki_id: str, project_id: str) -> ADOResponse:
+        """Get's the current version of a wiki page. This returns a required parameter for updating a wiki page.
+
+        https://docs.microsoft.com/en-us/rest/api/azure/devops/wiki/pages/get%20page%20by%20id?view=azure-devops-rest-6.1
+
+        :param page_id: Wiki page ID
+        :param wiki_id: Wiki ID or Wiki name
+        :param str project_id: The ID of the project
+
+        :returns: The ADO response with the data in it
+        """
+
+        self.log.debug(f"Get wiki page: {page_id}")
+        request_url = (
+            self.http_client.api_endpoint(is_default_collection=False, project_id=project_id)
+            + f"/wiki/wikis/{wiki_id}/pages/{page_id}?api-version=6.1-preview.1"
+        )
+        response = self.http_client.get(request_url)
+        self.http_client.validate_response(response)
+        return response.headers["ETag"]
```

Comment:
I'd make this a `.get` so that we can throw the appropriate exception. Might be worth checking `eTag` as well if `ETag` is missing since that's what the docs say they'll set (even though the spec is `ETag`).

Reply:
Turns out that the `.headers` property on a `Response` in the `requests` library is a `CastInsensitiveDict` already, so no need to check other casing.

Reply:
Good point 👍 Done